src/components/Header.vue

```diff
@@ -1,6 +1,12 @@
 <template>
   <v-app-bar :elevation="1">
     <v-app-bar-nav-icon @click.stop="showDrawerMenu = !showDrawerMenu"></v-app-bar-nav-icon>
+    <div style="margin-top: 8px; margin-right: 16px;">
```

Comment:
why a custom component ? we use vuetify everywhere, they have a [switch component](https://vuetifyjs.com/en/components/switches/#usage)

Reply:
Alright, I'll adjust the position by placing it on the far right of the navigation bar. Additionally, I wasn't aware of the custom component, but I'll make the required modifications.

Reply:
In Settings, honestly. Honor the browser default, and memorized in settings

Reply:
Sorry @teolemon, but I didn't understand what does that mean? Can you please elaborate?

Reply:
1. show the Dark mode settings in the settings screen
2. 3 Options: Dark mode, Light Mode, Follow system settings
3. By default, it follows system settings, as exposed by the browser https://css-tricks.com/a-complete-guide-to-dark-mode-on-the-web/

Reply:
Okay, I will make the necessary changes. Thanks for elaborating!